### DISCUSSION TITLE
How to get the title bar like in the demo video?

### DISCUSSION THREAD

**Discussion by stingray21** [kovidgoyal/kitty#4597] [Q&A] [answered]
Hello, 

I'm new to Kitty and trying to figure out how to configure it to my needs. And I was wondering how I can get the title bar to show all the information and style like in the demo video on https://sw.kovidgoyal.net/kitty/

![image](https://user-images.githubusercontent.com/4960916/151694871-971f2968-379b-4544-9e6f-50747b1984d0.png)

**Comment by page-down** [comment, marked as answer]
https://sw.kovidgoyal.net/kitty/kittens/panel/

You can start by looking at the github discussion list, there is already a discussion on the first page (as of the time of posting).

**Reply by stingray21** [reply]
Thank you! My apologizies, I don't know how I missed that
